frontend/src/pages/UserProfileConfiguration/ChangeUserEmail.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect, ChangeEvent } from 'react';
+import { CheckUserAuth } from 'pages/CheckUserAuth';
+import { Navbar } from '../../components/Navbar';
+import { useCookies } from 'react-cookie';
+import { ChangeUserMail } from '../../Utils/Utils';
+
+function ChangeUserEmail() {
```

Comment:
I think it would be great for this component to populate data of current email. 
User should be informed what his current email address. 

So there should be another one input, disabled for changes, with current user email.